api/ssi-bridge/src/routes/verification/tests/revoke-vc.test.ts

```diff
@@ -362,44 +354,6 @@ describe('test authentication routes', () => {
 			expect(loggerSpy).not.toHaveBeenCalled();
 			expect(nextMock).not.toHaveBeenCalled();
 		});
-
-		it('identity is already revoked', async () => {
```

Comment:
The implementation has changed. Before we were able to check this. Now we don't get this information anymore since `revokeCredentials` does not provide if the credential was already revoked or not anymore on sdk side